patches/minecraft/net/minecraft/world/chunk/Chunk.java.patch

```diff
@@ -215,7 +215,7 @@
          i = MathHelper.func_76125_a(i, 0, this.field_76645_j.length - 1);
          j = MathHelper.func_76125_a(j, 0, this.field_76645_j.length - 1);
  
-@@ -997,6 +100,8 @@
+@@ -997,6 +1000,8 @@
```

Comment:
It doesn't appear that you changed anything with this patch. When I had last asked @mezz about situations like this he had said to not commit patches like these.

Reply:
Yes, since this PR does not touch `Chunk.java` it should not include this file.